File: Java/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    protected Integer passengenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("License: " + license + " Name Driver: " + driver.name + " Passengers: " + passengenger);
    }

    public Integer getPassenger(){
        return passengenger;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
            this.passengenger = passenger;
        }
        else {
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
}

File: PHP/Car.php
<?php

    class Car {
        public $id = integer;
        public $license = string;
        public $driver = string;
        public $passenger = integer;
    }

    public function __construct($license, $driver)
    {
        $this->_license = $license;
        $this->_driver = $driver;
    }

    public function setPassengers($passenger)
    {
        $this->_passenger = $passenger;
    }

    public function getLicense()
    {
        return $this->_license;
    }

    public function getDriver()
    {
        return $this->_driver;
    }

?>